crates/adapter/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "viceroy-component-adapter"
+version = "0.0.0"
+edition = "2021"
+publish = false
+
+[dependencies]
+wasi = { workspace = true }
+wit-bindgen-rust-macro = { workspace = true }
+byte-array-literals = { workspace = true }
+bitflags = { workspace = true }
+
+[build-dependencies]
+wasm-encoder = { workspace = true }
+object = { workspace = true }
+
+[lib]
+test = false
+crate-type = ["cdylib"]
+name = "wasi_snapshot_preview1"
```

Comment:
This was to avoid renaming all the existing preview1 wrapper functions from the adapter, as it is a fork of the wasmtime crate. I think it's reasonable to rename it, but it will touch every function name, perhaps making it a little less obvious what the connection to the original repo is.

Reply:
I've renamed the wasm module, but internally still use the module name `wasi_snapshot_preview1` to avoid having to change the `export_name` of all the inherited functions from the original adapter. I put a big note in the only place where this happens, everywhere else uses the name `viceroy_component_adapter` with variations on underscores or hypens as required by tooling.